fetch-quotes.test.js

```diff
@@ -0,0 +1,21 @@
+const { fetchQuotes } = require('./fetch-quotes');
+const request = require('superagent');
+
+describe('fetchQuotes function', () => {
+    it('returns a single quote with the correct format', async () => {
+        const quote = {
+            character: "Bender",
+            quote: "Bite my shiny metal ass.",
+            image: "https://res.cloudinary.com/dzxqhkyqd/image/fetch/c_scale,w_500/https://res.cloudinary.com/dzxqhkyqd/image/upload/v1552429540/bender.png"
+        }
+
+        const fetchedQuote = await fetchQuotes();
+        expect(fetchedQuote)
+            .toEqual(
+                {
+                    character: expect.any(String),
```

Comment:
nice work incorporating the expect.any().  Jest is filled with so many useful way to test and will come in handy for more robust endpoints :)